### DISCUSSION TITLE
Geometry definition

### DISCUSSION THREAD

**Discussion by Lin-XX-01** [CERN/TIGRE#565] [Q&A]
Hi Ander,
I would like to ask whether the base coordinate system defined in the geometry file created in the tigre toolkit is on the ray source or the object to be measured, and if it is the object, does the base coordinate system follow the rotation of the object?

**Comment by AnderBiguri** [comment]
Hi, 
I am not sure I understand what you mean. Can you clarify? 
The coordinate system does not rotate no. Have you checked the paper/demo1 for the figure of the coordinate system?

**Comment by Lin-XX-01** [comment]
Hi Ander,
I'm sorry I didn't express my question clearly, I've looked at the demo1 example program again and my understanding is that the source point o of the fixed coordinate system is in the plane where the centre of the object is located, not fixed at the centre of the object, and then the rotational geometry of the object is determined by setting the object's offset geo.offOrigin relative to the base coordinate system right?

**Reply by AnderBiguri** [reply]
More or less, maybe yes. in other words:

The object is centered at (0,0,0), and the rotational axes is (0,0,1). Of you change geo.offOrigin you change the location of the center of the object, you shift the object wrt the axes. Rotation is still around (0,0,1).

Maybe another question from me: what are you trying to do? Maybe I can exain better if I know what the goal is

**Comment by Lin-XX-01** [comment]
Ok, this two scanning methods is what I want to achieve, is it possible to achieve this by modifying the scanning geometry parameters in the tigre toolkit?
![image](https://github.com/CERN/TIGRE/assets/174866719/fb8a98cd-bf0e-4791-b35c-4eb3138e934d)

**Reply by AnderBiguri** [reply]
Yes it is indeed. What you really want is to use the Euler angles of rotation to define your rotation. Check the demo for arbitrary axis of rotation to do (a). To do (b) you'd need to add to (a) a detector in-place rotation, there is also a demo for that.

**Reply by Lin-XX-01** [reply]
Ok, thank you very much for answering my query, I will go and study the relevant sample programmes.

**Comment by Lin-XX-01** [comment]
Hi Ander,
I would like to ask you some questions about demo18, for this kind of geometric scanning method where the axis of rotation can be deflected arbitrarily, I see that you point out that FDK will not work, does this mean that this kind of scanning method can't be rebuilt with FDK but only iterative reconstruction?
![image](https://github.com/user-attachments/assets/ac137357-c507-4734-8277-2d18bb1cfc7d)

**Reply by AnderBiguri** [reply]
That's correct. You can still try if you want, but the math of FDK assumes a circular trajectory, so it won't produce good results likely

**Reply by Lin-XX-01** [reply]
Well, thank you very much for your reply and have a nice life!
